src/Microsoft.DocAsCode.Common/FileAbstractLayer/PathMapping.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Common
+{
+    using System;
+    using System.Collections.Immutable;
+
+    public struct PathMapping : IEquatable<PathMapping>
+    {
+        public PathMapping(RelativePath logicalPath, string physicalPath)
```

Comment:
when used for source, first is source relative path, second is physical path;
when used for target, first is target relative path, second is physical path.

---
In reply to: [90155901](https://github.com/dotnet/docfx/pull/926#discussion_r90155901) [](ancestors = 90155901)